src/bindings/python/tests_compatibility/test_inference_engine/test_InferRequest.py

```diff
@@ -283,6 +283,7 @@ def callback(self, status):
     del ie_core
 
 
+@pytest.mark.skip(reason="Plugin API 2.0 has the different behavior for request.wait().")
```

Comment:
can we just update the test according to new behavior?

Reply:
Considered other hw pluginw still are legacy plugin api, maybe there will be some problem for them?

Reply:
I suppose new behavior is more relaxed than old one. So, if we stick to new one, old plugins should also pass the tests.

Reply:
New api is more relaxed than old one, but legacy python API has very strict checking, which cause such failure. If we can loose legacy python api checking, such test can be pass.
Anyhow, does it make sense to modify these python api tests?